x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/SecurityServerTransportInterceptor.java

```diff
@@ -79,6 +81,8 @@ public class SecurityServerTransportInterceptor implements TransportInterceptor
     static final Set<String> REMOTE_ACCESS_ACTION_ALLOWLIST;
     static {
         final Stream<String> actions = Stream.of(
+            REMOTE_CLUSTER_HANDSHAKE_ACTION_NAME,
+            RemoteClusterNodesAction.NAME,
             TransportService.HANDSHAKE_ACTION_NAME,
```

Comment:
Yes. The allowlist is becoming asymmetric. In fact, it was asymmetric before this PR because QC side should never see proxy'd actions, which are also rewritten after the interceptor. I added a discussion item for this week's sync to see whether we should rop the allowlist on the QC side.